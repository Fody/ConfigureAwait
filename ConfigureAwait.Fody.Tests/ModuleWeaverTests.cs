using System.Reflection;
using ConfigureAwait.Fody;
using Fody;
#pragma warning disable 618

public partial class ModuleWeaverTests
{
    static TestResult testResult;

    static ModuleWeaverTests()
    {
        Assembly.Load("xunit.assert");
        var weavingTask = new ModuleWeaver();
        testResult = weavingTask.ExecuteTestRun("AssemblyToProcess.dll");
#if NET46
#if DEBUG
        ApprovalTests.Namers.NamerFactory.AsEnvironmentSpecificTest(() => "Debug");
#else
        ApprovalTests.Namers.NamerFactory.AsEnvironmentSpecificTest(() => "Release");
#endif
#endif
    }
}