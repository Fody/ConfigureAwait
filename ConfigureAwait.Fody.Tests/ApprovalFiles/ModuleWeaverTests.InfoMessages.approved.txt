Info[0] = 	Removing reference to 'ConfigureAwait'.
